server/scripts/clear-data.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-console */
+require('dotenv').config({ path: '../.env' });
+const { dbClient } = require('../db');
+const { databaseCollections } = require('../db/schema');
```

Comment:
`Require`s should be replaced with imports, and calls to `dotenv` should be replaced with `import './load-env';` at the top of the file.

Reply:
Oh yes, my bad, didn't handle the merge